### PR TITLE
Chapter 1, Lesson 4 homework.

### PR DESCRIPTION
**Pull request by vytautas-sablinskas**
Checker in VS doesn't work, so I'll check it by using PR.

### CODE REVIEW

**Comment by vytautas-sablinskas** [comment]
I have no clue honestly what to do right now, probably will watch Lesson 5 and then try to comeback because I'm stuck on this for long time already and Float checks don't show if they're correct or wrong.

**Comment by vytautas-sablinskas** [comment]
Done all checks, I think it should be alright now.

File: Src/BootCamp.Chapter/Lesson4.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson4.cs]: The body of for loop could be moved into a function. That would leave you with two options then:
A- just call the same function twice
B- do what you do, but in a body call a function instead.

File: Src/BootCamp.Chapter/Lesson4.cs [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                return -1;
+            }
+            return number;
+        }
+        public static float CalculateBmi(float weight, float height)
+        {
+            bool isValidWeight = true;
+            bool isValidHeight = true;
+
+            if (height <= 0)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson4.cs]: Can be simplified:
```cs
string error = "";
if(weight <= 0)
{
 error = ...;
}
else if(height <= 0)
{
  error += "";
}
if(string.IsNullOrEmpty(error))
{
  print(error);
  return -1;
}
else
{
  return bmi;
}
```

**Review by Almantask — CHANGES_REQUESTED** [review]
A bit overly complex validation

**Comment by vytautas-sablinskas** [comment]
Okay, so I redid it a bit. Now I added another function AskingInformation and called it in Demo function. 
Then I tried simplifying this thing, but not sure if I got it right. It's pretty hard to understand this, but I hope I can get better at it.

**Review by Almantask — APPROVED** [review]
Looks better 😄

**Pull request closed without merging**